core/src/main/java/org/apache/spark/api/plugin/PluginRpcEndpoint.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.api.plugin;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.apache.spark.SparkContext;
```

Comment:
Hi, @vanzin . Could you fix the `dev/lint-java`? `Collections` / `Map` / `SparkContext` seems to be unused in this interface.
```
[ERROR] src/main/java/org/apache/spark/shuffle/sort/io/LocalDiskShuffleDriverComponents.java:[24,8] (imports) UnusedImports: Unused import - org.apache.spark.SparkEnv.
38
[ERROR] src/main/java/org/apache/spark/shuffle/api/ShuffleExecutorComponents.java:[21,8] (imports) UnusedImports: Unused import - java.util.Map.
39
[ERROR] src/main/java/org/apache/spark/shuffle/api/ShuffleDriverComponents.java:[20,8] (imports) UnusedImports: Unused import - java.util.Map.
40
[ERROR] src/main/java/org/apache/spark/api/plugin/PluginRpcEndpoint.java:[20,8] (imports) UnusedImports: Unused import - java.util.Collections.
41
[ERROR] src/main/java/org/apache/spark/api/plugin/PluginRpcEndpoint.java:[21,8] (imports) UnusedImports: Unused import - java.util.Map.
42
[ERROR] src/main/java/org/apache/spark/api/plugin/PluginRpcEndpoint.java:[23,8] (imports) UnusedImports: Unused import - org.apache.spark.SparkContext.
```